src/backup_camera.cpp

```diff
@@ -99,12 +101,12 @@ bool BackupCamera::init_graphics(SDL_Renderer* renderer)
 //Used for any "Updates" you need. Currently there is only one "camera" within this class.
 bool BackupCamera::BackupCamera::update()
 {
-    //music_bar_one_->update(graphics_handler_);
+
     return camera_one_->update(graphics_handler_);
 }
 
 
-bool BackupCamera::process_events()
+int BackupCamera::process_events()
```

Comment:
Since multiple events can happen at once, i think this function would make more sense to return a vector of the events. This will also make the main look better, where when you process events instead of writing a while loop to specifically catch one event, you can instead check if any events were returned and then deal with all the events individually.